mono/mini/aot-runtime.c

```diff
@@ -3313,9 +3315,11 @@ mono_aot_find_jit_info (MonoDomain *domain, MonoImage *image, gpointer addr)
 
 	nmethods = amodule->info.nmethods;
 
+#ifndef ENABLE_INTERPRETER
```

Comment:
This looks wrong.
It means we can't enable the interpreter on desktop and have working (non-full)AOT.